proposals/interledger/README.md

```diff
@@ -160,4 +160,4 @@ This data can be used by the website to consult a back-end API to further verify
 
 As `interledger` is an open payment method it will require explicit implementation by browsers. This is an opportunity for browsers to assist in securing the payment by implementing the Payment Pointer protocol in the browser and simply returning the result of this to the Payment Handler.
 
-In this way the browser can perform additional security steps such as filtering the receiver endpoint URL against known malicious origins, ensuring that the certificates used by the receiver endpoint meet strict security standards or enforcing further security measures such as requiring that the origin of the receiver endpoint is the same as (or explicitly trusted by) the origin of the Payment Request API caller.
\ No newline at end of file
+In this way the browser can perform additional security steps such as filtering the receiver endpoint URL against known malicious origins, ensuring that the certificates used by the receiver endpoint meet strict security standards or enforcing further security measures such as requiring that the origin of the receiver endpoint is the same as (or explicitly trusted by) the origin of the Payment Request API caller.
```

Comment:
Not sure why it says this line changed, maybe github's online editor changed the newline character.